x/mint/keeper/keeper.go

```diff
@@ -137,56 +143,99 @@ func (k Keeper) GetProportions(ctx sdk.Context, mintedCoin sdk.Coin, ratio sdk.D
 	return sdk.NewCoin(mintedCoin.Denom, mintedCoin.Amount.ToDec().Mul(ratio).TruncateInt())
 }
 
+const (
+	// TODO (set this!)
```

Comment:
This todo feels like it could get lost - can you file a ticket here?

Reply:
tracking in TEST-178